templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
+  </form>
+{% endblock %}
```

Comment:
It's better to always end files with a newline character. You can read why this is the case [here](https://stackoverflow.com/a/729795).
Fix it for other files as well.

Reply:
oh, sorry forgot about it, thank you